tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -14,11 +14,34 @@ Do one of the following actions to access the Runtime Resources Visibility plug-
   install [Tanzu Application Platform GUI separately](../install-tap-gui.md)
 - Review the section [If you have a metrics server](#metrics-server)
 
-## <a id="metrics-server"></a> If you have a metrics server
+## <a id="metrics-server"></a> If You Have a Metrics Server
 
 By default, the Kubernetes API does not attempt to use any metrics servers on your clusters.
 To access metrics information for a cluster, set `skipMetricsLookup` to `false` for that
-cluster in the `kubernetes` section of `app-config.yaml`.
+cluster in the `kubernetes` section of `app-config.yaml`. Below is an 
+
+```
+tap_gui:
+  ... // Existing configuration
```

Comment:
My preference would be that we don't give users yaml with syntax errors. Even in docs. My suggestion would be to do something like
```yaml
tap_gui:
  # ... existing configuration
  app_config:
    # ... existing configuration
    kubernetes:
```
and so on

Reply:
Love it. I was looking for that syntax and to see if there was a standard way we were already doing that.